app/views/govuk_publishing_components/components/docs/breadcrumbs.yml

```diff
@@ -3,6 +3,8 @@ description: "Navigational breadcrumbs, showing page hierarchy"
 body: |
   Accepts an array of breadcrumb objects. Each crumb must have a title and a URL.
   Links have tracking data but links to the homepage (any link with a url of "/") will be tracked separately as `homeLinkClicked`
+
+  Note: Only the first and last (or parent) item in the breadcrumb will be displayed on a page width less than 640px.
```

Comment:
Can we use a breakpoint (e.g. tablet) instead of a fixed value in this sentence? It may be that 640px is not right, as we don't have a breakpoint with this value.

Reply:
updated not to refer to tablet breakpoint not a specific value